### ISSUE TITLE
Wireguard and H/3

### ISSUE THREAD

**Issue by Avamander** [yrutschle/sslh#415]
I'm trying to differentiate between Wireguard and H/3 (QUIC) connections on UDP.

But even with this config (that is quite accepting):
```
     { name: "wireguard"; host: "localhost"; is_udp: true; port: "1443"; log_level: 1 },
     { name: "regex"; host: "localhost"; is_udp: true; port: "8443"; regex_patterns: [ "\x00\x00\x00" ], log_level: 1 }
```

Quite a few H/3 connections fail. I've also tried reordering the protocols and to manually match Wireguard. I still can't achieve a reliable combination between the two. I'm using nginx as the H/3 server configured as their documentation states.

Is it possible to redirect just Wireguard and not touch anything else on that port?

**Comment by yrutschle** [comment]
Yes, you can use `anyprot` instead of `regex`, and everything that's not matched by other rules will go there.

**Comment by Kobaxidze256** [comment]
You can look into the built-in wireguard probe and also check this<https://github.com/yrutschle/sslh/issues/466#issuecomment-4531098063> out
